src/app/public/src/modules/image/image.component.scss

```diff
@@ -10,3 +10,8 @@
   max-width: 100%;
   height: auto;
 }
+
+.stache-image-caption {
+  margin: 0.5em 1em 0.5em 1em;
+  text-align: center;
```

Comment:
Moving to left align, we can us padding rather than margin here.  `padding: 0.5em 0;`